packages/react/.storybook/preview.tsx

```diff
@@ -21,7 +21,7 @@ export const parameters = {
 // so that resources created in MockFetchClient#initMockRepo have
 // consistent timestamps between storybook runs
 const clock = createGlobalTimer();
-const medplum = new MockClient();
+const medplum = new MockClient({ strictMode: false });
```

Comment:
We need to disable `strictMode` in `storybook` since we have some searches that reference types that we don't have the `StructureDefinition` or `SearchParameter`s for